README.md

```diff
@@ -1,2 +1,146 @@
-# eslint-config-pat
-Shareable ESLint config for PAT projects
+# @nordcloud/eslint-config-pat
+
+A TypeScript ESLint ruleset designed for Nordcloud's Platform & Tools
+
+## Implementation
+
+- **Monorepo friendly:** The `@nordcloud/eslint-config-pat` package has direct dependencies on all the ESLint plugins
+  that it needs. This avoids encumbering each consuming project with the obligation to satisfy a peer dependencies.
+  It also ensures that the installed plugin versions were tested for compatibility together.
+
+- **Designed for Prettier:** The `@nordcloud/eslint-config-pat` ruleset is designed to be used together with
+  the [Prettier](https://prettier.io/) code formatter.
+  Prettier avoids frivolous debates: its defaults have already been debated
+  at length and adopted by a sizeable community.
+
+- **Minimal configuration:** To use this ruleset, your **.eslintrc.js** will need to choose one **"profile"**
+  and possibly one or two **"mixins"** that cover special cases
+
+## Getting started in 3 steps
+
+Applying the ruleset to your project is quick and easy. You install the package, then create an **.eslintrc.js** file
+and select an appropriate project profile. Optionally you can also add some "mixins" to enable additional rules.
+Let's walk through those three steps in more detail.
+
+### 1. Install the package
+
+To install the package, do this:
+
+```sh
+cd your-project-folder
+npm install -D eslint typescript prettier @nordcloud/eslint-config-pat
+```
+
+### 2. Choose one profile
+
+The ruleset currently supports three different "profile" strings, which select lint rules applicable for
```

Comment:
You are mentioning three different profiles but below there are only two enumerated.